controllers/hcpvaultsecretsapp_controller.go

```diff
@@ -80,14 +83,16 @@ var (
 // HCPVaultSecretsAppReconciler reconciles a HCPVaultSecretsApp object
 type HCPVaultSecretsAppReconciler struct {
 	client.Client
-	Scheme                      *runtime.Scheme
-	Recorder                    record.EventRecorder
-	SecretDataBuilder           *helpers.SecretDataBuilder
-	HMACValidator               helpers.HMACValidator
-	MinRefreshAfter             time.Duration
-	referenceCache              ResourceReferenceCache
-	GlobalTransformationOptions *helpers.GlobalTransformationOptions
-	BackOffRegistry             *BackOffRegistry
+	Scheme                              *runtime.Scheme
+	Recorder                            record.EventRecorder
+	SecretDataBuilder                   *helpers.SecretDataBuilder
+	HMACValidator                       helpers.HMACValidator
+	MinRefreshAfter                     time.Duration
+	referenceCache                      ResourceReferenceCache
+	GlobalTransformationOptions         *helpers.GlobalTransformationOptions
+	BackOffRegistry                     *BackOffRegistry
+	CleanupOrphanedShadowSecretInterval time.Duration
```

Comment:
nit: does this need to be member of this structure? Could we maybe pass it in when calling SetupWithManager()?